### PR TITLE
fix(dummy_perception): fix to use launch at perception launch

### PR DESCRIPTION
**Pull request by taikitanaka3**
## Related Issue(required)

- improvements

## Description(required)

laserscan to occupancy grid refer to perception launch 
fix angle increment according to https://github.com/tier4/autoware_launch/pull/220

```
    -  <include file="$(find-pkg-share laserscan_to_occupancy_grid_map)/launch/laserscan_to_occupancy_grid_map.launch.py">
    + <include file="$(find-pkg-share tier4_perception_launch)/launch/occupancy_grid_map/occupancy_grid_map.launch.py">
```

@kenji-miyake 
is this change ok to use perception_launch?

## Review Procedure(required)
launch psim and see the behavior is the same
Screenshot from 2022-03-01 10-19-16

## Related PR(optional)

https://github.com/tier4/autoware_launch/pull/220

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [x] Read [commit-guidelines][commit-guidelines]
- [x] Assign PR to reviewer

If you are adding new package following items are required:

- [ ] Documentation with description of the package is available
- [ ] A sample launch file and parameter file are available if the package contains executable nodes

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[commit-guidelines]: https://www.conventionalcommits.org/en/v1.0.0/
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

**Comment by kenji-miyake** [comment]
> is this change ok to use perception_launch?

@taikitanaka3 Actually no. `autoware.universe` can't depend on https://github.com/tier4/autoware_launch.
Please explain why you need this.

**Review by yukkysaito — CHANGES_REQUESTED** [review]
Please keep tier4_perception_launch

File: simulator/dummy_perception_publisher/launch/dummy_perception_publisher.launch.xml [hunk 1]
```diff
@@ -55,7 +55,7 @@
     <group>
       <push-ros-namespace namespace="occupancy_grid_map"/>
       <!-- Occupancy Grid -->
-      <include file="$(find-pkg-share laserscan_to_occupancy_grid_map)/launch/laserscan_to_occupancy_grid_map.launch.py">
+      <include file="$(find-pkg-share perception_launch)/launch/occupancy_grid_map/occupancy_grid_map.launch.py">
```
Comment (yukkysaito) [inline simulator/dummy_perception_publisher/launch/dummy_perception_publisher.launch.xml:58]:
```suggestion
      <include file="$(find-pkg-share tier4_perception_launch)/launch/occupancy_grid_map/occupancy_grid_map.launch.py">
```

File: simulator/dummy_perception_publisher/launch/dummy_perception_publisher.launch.xml [hunk 2]
```diff
@@ -70,13 +70,13 @@
       <!-- tracking module -->
       <group>
         <push-ros-namespace namespace="tracking"/>
-        <include file="$(find-pkg-share tier4_perception_launch)/launch/object_recognition/tracking/tracking.launch.xml">
+        <include file="$(find-pkg-share perception_launch)/launch/object_recognition/tracking/tracking.launch.xml">
```
Comment (yukkysaito) [inline simulator/dummy_perception_publisher/launch/dummy_perception_publisher.launch.xml:73]:
```suggestion
        <include file="$(find-pkg-share tier4_perception_launch)/launch/object_recognition/tracking/tracking.launch.xml">
```

File: simulator/dummy_perception_publisher/launch/dummy_perception_publisher.launch.xml [hunk 3]
```diff
@@ -70,13 +70,13 @@
...
         </include>
       </group>
       <!-- prediction module -->
       <group>
         <push-ros-namespace namespace="prediction"/>
-        <include file="$(find-pkg-share tier4_perception_launch)/launch/object_recognition/prediction/prediction.launch.xml">
+        <include file="$(find-pkg-share perception_launch)/launch/object_recognition/prediction/prediction.launch.xml">
```
Comment (yukkysaito) [inline simulator/dummy_perception_publisher/launch/dummy_perception_publisher.launch.xml:79]:
```suggestion
        <include file="$(find-pkg-share tier4_perception_launch)/launch/object_recognition/prediction/prediction.launch.xml">
```

File: simulator/dummy_perception_publisher/launch/dummy_perception_publisher.launch.xml [hunk 4]
```diff
@@ -92,7 +92,7 @@
     <!-- traffic light module -->
     <group if="$(var use_traffic_light_recognition)">
       <push-ros-namespace namespace="traffic_light_recognition"/>
-      <include file="$(find-pkg-share tier4_perception_launch)/launch/traffic_light_recognition/traffic_light.launch.xml"></include>
+      <include file="$(find-pkg-share perception_launch)/launch/traffic_light_recognition/traffic_light.launch.xml"></include>
```
Comment (yukkysaito) [inline simulator/dummy_perception_publisher/launch/dummy_perception_publisher.launch.xml:95]:
```suggestion
      <include file="$(find-pkg-share tier4_perception_launch)/launch/traffic_light_recognition/traffic_light.launch.xml"></include>
```

**Review by yukkysaito — APPROVED** [review]
LGTM

**Review by yukkysaito — APPROVED** [review]
LGTM

**Pull request merged**